nala/features/mas/benchmark/benchmark.test.js

```diff
@@ -0,0 +1,50 @@
+import { expect, test } from '@playwright/test';
+import { features } from './benchmark.spec.js';
+import BenchmarkPage from './benchmark.page.js';
+
+const miloLibs = process.env.MILO_LIBS || '';
+
+test.beforeEach(async ({ page, browserName }) => {
+  test.skip(browserName !== 'chromium', 'Not supported to run on multiple browsers.');
+  if (browserName === 'chromium') {
+    await page.setExtraHTTPHeaders({ 'sec-ch-ua': '"Chromium";v="123", "Not:A-Brand";v="8"' });
+  }
+  // disabling cache
+  await page.route('**', (route) => route.continue());
+});
+
+test.describe('Benchmark feature test suite', () => {
+  test(`${features[0].name},${features[0].tags}`, async ({ page, baseURL }) => {
+    const testPage = `${baseURL}${features[0].path}${miloLibs}`;
```

Comment:
I do have a concern including this in Milo nala tests.
If freya or wcs cdn will be causing slower render it will fail Milo nala tests and block developers until someone will comment this test out.
We could try it out and if we trues in Freya/WCS stability it would work, however I think we should notify people so that they are aware about this test having external dependencies (it's ok for Nala tests)
And if this test will be flaky we will have to take it out and maybe run it only manually/before release.
We should make sure we don't create stress for Milo community with this

Reply:
I agree, this is why i added alignement with network capabilities. For the rest items should be cached, so this is CDN response time for constant time we are measuring and this should not change much.
I thought about not adding this systematically and/or raise acceptance bar, but i think this will defeat the purpose of us guaranteeing cards performances